### PR TITLE
Adds state abbreviation validation

### PR DESCRIPTION
**Pull request by tom-con**
Changes example.env to match other repos

### CODE REVIEW

**Comment by bearfact** [comment]
:+1:

File: tests/joi/index.test.js [hunk 1]
```diff
@@ -45,5 +45,31 @@ describe('joiUtils', () => {
...
+			expect(() => Joi.assert("CM", schema)).toThrow();
+			expect(() => Joi.assert("PR", schema)).toThrow();
+      expect(() => Joi.assert("QQ", schema)).toThrow();
+      expect(() => Joi.assert("ny", schema)).toThrow();
+			expect(() => Joi.assert("Alabama", schema)).toThrow();
+			expect(() => Joi.assert("", schema)).toThrow();
+      expect(() => Joi.assert(12, schema)).toThrow();
+      expect(() => Joi.assert([], schema)).toThrow();
+    });
+    test('error is thrown if schema is tested against valid values', () => {
```
Comment (siennamw) [inline tests/joi/index.test.js]:
```suggestion
    test('error is not thrown if schema is tested against valid values', () => {
```

File: joi/index.js [hunk 2]
```diff
@@ -14,6 +14,12 @@ const joiSimpleDate = () => {
 	return Joi.string().regex(/^\d{4}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])$/);
 };
 
+const joiStateAbbreviation = () => {
+	return Joi.string().regex(
+		/^(?:(A[KLRZ]|C[AOT]|D[CE]|FL|GA|HI|I[ADLN]|K[SY]|LA|M[ADEINOST]|N[CDEHJMVY]|O[HKR]|P[A]|RI|S[CD]|T[NX]|UT|V[AT]|W[AIVY]))$/
```
Comment (siennamw) [inline joi/index.js]: what's your thinking behind using a capturing group wrapped in a non-capturing group?

File: tests/joi/index.test.js (hunk 1 above)
Comment (tom-con) [reply tests/joi/index.test.js]: nice

File: joi/index.js (hunk 2 above)
Comment (tom-con) [reply joi/index.js]: Ah didn't even notice the nested capturing group. I'll remove. Totally copied this from somewhere.

**Comment by siennamw** [comment]
👍

**Comment by tom-con** [comment]
👍

**Pull request merged**